topgun/k8s/baggageclaim_drivers_test.go

```diff
@@ -37,6 +39,41 @@ var _ = Describe("baggageclaim drivers", func() {
 			baggageclaimWorks("naive", UBUNTU)
 		})
 
+		Context("with a real btrfs partition", func() {
+			It("successfully recreates the worker", func() {
+				By("deploying concourse with ONLY one worker and having the worker pod use the gcloud disk and format it with btrfs")
+
+				setReleaseNameAndNamespace("real-btrfs-disk")
+
+				deployWithDriverAndSelectors("btrfs", UBUNTU,
+					"--set=persistence.enabled=false",
+					"--set=worker.additionalVolumes[0].name=concourse-work-dir",
+					"--set=worker.additionalVolumes[0].gcePersistentDisk.pdName=disk-topgun-k8s-btrfs-test",
+					"--set=worker.additionalVolumes[0].gcePersistentDisk.fsType=btrfs",
+				)
+
+				waitAllPodsInNamespaceToBeReady(namespace)
+
+				By("Creating the web proxy")
+				_, atcEndpoint := startPortForwarding(namespace, "service/"+releaseName+"-web", "8080")
```

Comment:
oh, I don't think this would compile - with the changes to allow running from within a k8s cluster not using port forwarding, this method doesn't exist anymore:

https://github.com/concourse/concourse/blob/649ac22d6012be6d2832ae2437596bd48380ef13/topgun/k8s/https_web_tls_termination_test.go#L62-L66

Reply:
Done! Thanks for catching that.